src/Symfony/Bundle/FrameworkBundle/DataCollector/CommandDataCollector.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DataCollector;
+
+use Symfony\Bundle\FrameworkBundle\Debug\CliRequest;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SignalRegistry\SignalMap;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+use Symfony\Component\VarDumper\Cloner\Data;
+
+class CommandDataCollector extends DataCollector
```

Comment:
```suggestion
final class CommandDataCollector extends DataCollector
```

I would also mark this class internal to be able to easily change it's public API in the feature, and make it independent from the HTTP layer.

Reply:
Done :).